pom.xml

```diff
@@ -32,46 +32,46 @@
 
 		<openMRSVersion>2.3.1</openMRSVersion>
 		<referencemetadataVersion>2.11.0-SNAPSHOT</referencemetadataVersion>
-		<appframeworkVersion>2.17.0-SNAPSHOT</appframeworkVersion>
```

Comment:
Why revert to a previous release?

Reply:
Thanks @dkayiwa, The previous version was 2.14. then it was updated to 2.16 because the last changes were made in 2.16.0